src/shared/views/Landing.tsx

```diff
@@ -102,6 +105,40 @@ const Landing: React.FunctionComponent<LandingProps> = ({
       });
   };
 
+  const saveAndDeprecate = (selectedOrg: Organization) => {
+    setFormBusy(true);
+
+    deprecateOrg(selectedOrg.label, selectedOrg.rev)
+      .then(
+        () => {
+          notification.success({
+            message: 'Organization deprecated',
+            duration: 2,
```

Comment:
Maybe these durations should be stored in config somewhere, with relevant constant names like NOTIFICATION_DURATION_DEFAULT = 2 and NOTIFICATION_DURATION_PERMANENT = 0

Reply:
Agreed, but to be honest, I think the whole thing needs to be refactored. There is a lot of code duplication + the way the UI presents this to the user is quite basic. So it's going to change a lot very soon I believe